src/components/gds-media-card/gds-media-card.scss

```diff
@@ -14,6 +10,7 @@
 
 .description {
   @include description;
+  margin: 0 24px;
```

Comment:
Just to help me understand the best practices, why is this style here and not in the `description` mixin?

Reply:
Aha :) Thanks for the explanation :D